tests/external/start.spec.js

```diff
@@ -21,18 +21,19 @@ describe('start script', function() {
             expect(rows.length).to.equal(1);
             expect(rows[0].id).to.equal(10);
             done();
-        });      
+        });
     });
 
-    it('starts http ping server', function(done) {        
+    it('starts http ping server', function(done) {
         var ping = {
-            url: 'http://' + server.ip + ':' + server.port + '/ping',
+            host: server.ip,
+            port: server.port,
+            path: '/ping',
             headers: {
```

Comment:
I probably don't understand something, but I'm not sure why you have to set the header here isntead of using the support request headers ...